strawberry/aiohttp/handlers/graphql_transport_ws_handler.py

```diff
@@ -21,7 +22,7 @@ def __init__(
         debug: bool,
         connection_init_wait_timeout: timedelta,
         get_context: Callable[..., Dict[str, Any]],
-        get_root_value: Any,
+        get_root_value: Callable[..., Optional[RootValue]],
```

Comment:
mmh, I think we don't need Optional here 🤔 (same below)

Reply:
I added Optional because of this implementation, is it wrong?

https://github.com/strawberry-graphql/strawberry/blob/2f02b4999007b06187bbe10ac68dc3f503d35853/strawberry/channels/handlers/ws_handler.py#L118-L119